dist/AbstractApi.js

```diff
@@ -45,16 +45,7 @@ class AbstractApi {
         }
         const root = this.apiRoot.endsWith('/') ? this.apiRoot : this.apiRoot + '/';
         const request = new Request(root + url, options);
-        let response;
-        try {
-            response = await fetch(request);
-        }
-        catch (e) {
-            if (handleLoading && !!this.loadingProvider) {
-                this.loadingProvider.onResolve();
-            }
-            return Promise.reject(e);
-        }
+        const response = await fetch(request);
```

Comment:
not sure if the handle loading provider will ever activate onResolve if fetch throws an error?